artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -907,8 +907,7 @@ public RoutingStatus route(final Message message,
       AddressInfo addressInfo = addressManager.getAddressInfo(address);
 
       if (bindingMove != null) {
-         context.clear();
-         context.setReusable(false);
+         context.clear().setReusable(false);
```

Comment:
really is that needed? having on two lines helps when running debuggers

Reply:
not needed, I'm just keeping how this is called everywhere else. I don't see it much different.. I was debugging it the whole day.

I can take it out from the diff.

Reply:
I reverted these two lines. no big deal.

will mark this section as resolved.